core/lib/dal/src/blocks_dal.rs

```diff
@@ -2319,6 +2320,30 @@ impl BlocksDal<'_, '_> {
             .context("delete_initial_writes_inner()")?;
         Ok(())
     }
+
+    /// Retrieves the blobs for a given batch
```

Comment:
please clearly mention what happens when a given batch didn't have blobs (for example for older protocol versions)